API.yaml

```diff
@@ -1131,6 +1131,68 @@ paths:
             application/json:
               schema:
                 type: string
+  /experiments/{expUUID}/outputs/table/{outputID}/tooltip:
+    post:
+      tags:
+      - Virtual Tables
+      summary: API to get virtual table tooltip
+      operationId: getTooltip
+      parameters:
+      - name: expUUID
+        in: path
+        description: The UUID of the experiment in the server
+        required: true
+        schema:
+          type : string
+          format: uuid
+      - name: outputID
+        in: path
+        description: The name of the table provider to query
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: Query parameters to fetch the timegraph tree.
```

Comment:
Update description? Also, I don't think the parameters mentioned below apply to this query, right?